content/en/docs/Architecture/cloud-init.md

```diff
@@ -143,6 +143,153 @@ When a Kairos boots it creates sentinel files in order to allow to execute cloud
 To execute a block using the sentinel files you can specify: `if: '[ -f "/run/cos/..." ]'`, for instance:
 
 
+## Default Kairos configs
+
+We have a set of default cloud-init configs that are shipped with the base image, and can be found in `/system/oem/`. These configs are executed during the various boot stages and can be overridden by user-provided configs in `/oem` or `/usr/local/cloud-config`.
+
+You can check the default configs in the [kairos-init repository](https://github.com/kairos-io/kairos-init/tree/main/pkg/bundled/cloudconfigs).
+
+
+## Overriding default configs
+
+You can override the default configs by creating a new file in `/oem` and adding your custom configuration. For example, if you want to override the [default sysctl configuration](https://github.com/kairos-io/kairos-init/blob/main/pkg/bundled/cloudconfigs/09_systemd_services.yaml#L7) provided by Kairos on the `boot` stage you would create a file under /oem with the following content:
```

Comment:
if adding the line number it might be good to link to a sha and not main in case it changes, not a blocker though